lib/components/Buttons/ButtonBase.tsx

```diff
@@ -22,6 +22,7 @@ export type ButtonBaseProps = {
   disabled?: boolean;
   /** Apply rounded corners */
   rounded?: boolean;
+  circle?: boolean;
```

Comment:
I assume circle and rounded are never both true. So how about:
```suggestion
  shape?: "default" | "rounded" | "circle";
```

Where 'default' is the fallback if no prop is provided.
This will require you to update ~19 instances where the `rounded` prop is used.

Reply:
Nice, that update with this 👍